spec.html

```diff
@@ -28965,13 +28965,27 @@ <h1>
         <h1>
           HostEnsureCanCompileStrings (
             _calleeRealm_: a Realm Record,
+            _goal_: a grammar symbol,
+            _bodyText_: an ECMAScript language value,
           ): either a normal completion containing ~unused~ or a throw completion
         </h1>
         <dl class="header">
           <dt>description</dt>
           <dd>It allows host environments to block certain ECMAScript functions which allow developers to interpret and evaluate strings as ECMAScript code.</dd>
         </dl>
         <p>The default implementation of HostEnsureCanCompileStrings is to return NormalCompletion(~unused~).</p>
+
+        <emu-note>
+          <p>
+            _goal_ is the nonterminal which will be used to parse _bodyText_.
+            For example, if called via %Function% it might be the grammar symbol |FunctionBody[~Yield, ~Await]| and from %eval% it might be the grammar symbol |Script|.
+          </p>
+        </emu-note>
```

Comment:
Since the Note is about a parameter, it should maybe go at the top, so that it appears after the para that introduces the parameters.

Reply:
Agreed. I guess it probably doesn't need to be a Note, either, since it's just a description of the parameter.